example/blzpack.c

```diff
@@ -43,6 +45,13 @@
  * All values in the header are stored in network order (big endian, most
  * significant byte first), and are read and written using the read_be32()
  * and write_be32() functions.
+ *
+ * Note that without padding, a new block header starts directly after
```

Comment:
What is the use-case for padding?

If there is no requirement for this, I feel it complicates things without improving the example.

Reply:
Honestly it's just neater to me. It may also (very slightly) improve performance, but my reason is order.

Reply:
Okay, I'd rather not add this then, sorry.

Reply:
All right, removed from the latest commits.